web/virtual_lab/static/virtual_lab/js/chemistry/titration.js

```
@@ -0,0 +1,117 @@
+// static/virtual_lab/js/chemistry/titration.js
+
+const beakerCanvas = document.getElementById('titration-canvas');
+const bctx         = beakerCanvas.getContext('2d');
+const dropCanvas   = document.getElementById('drop-canvas');
+const dctx         = dropCanvas.getContext('2d');
+
+let titrantVolume = 0, loopID;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)